.github/workflows/create-release.yml

```diff
@@ -0,0 +1,29 @@
+# Github Action to create a release with goreleaser
+name: Create Release
+on:
+  push:
+    # Sequence of patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0, v20.15.10
+
+jobs:
+  goreleaser:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        name: Set up Go
+        uses: actions/setup-go@v3
+      -
+        name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v4
+        with:
+          distribution: goreleaser
```

Comment:
this is awesome! 
Could you please kindly add a comment here or in Readme.md where to locate the new release image?

Reply:
Yes, I'll add something to the README.md but if its ok I'll do it in a follow up PR.